site/content/en/docs/Installation/Creating Cluster/gke.md

```diff
@@ -121,6 +126,19 @@ Flag explanations:
 * node-labels: The Kubernetes labels to automatically apply to nodes in this node pool.
 * num-nodes: The Agones system controllers only require a single node of capacity to run. For faster recovery time in the event of a node failure, you can increase the size to 2.
 
+_Optional_: Create a dedicated Windows node pool for game servers. If you need to run game servers on Windows you'll
```

Comment:
Please surround this with 

{{% feature publishVersion="1.14.0" %}}
...
{{% /feature %}}

so that it goes live with the next release (when we actually have an agones sdkserver that will run on windows nodes).